tests_react_native/sendPushNotification.js

```diff
@@ -0,0 +1,53 @@
+var admin = require('firebase-admin')
+
+var serviceAccount = require('./notifee-admin.json')
```

Comment:
believe it or not, I have not had bad luck with a mostly open firebase project in the invertase space, but I've been using it for https://github.com/invertase/react-native-firebase-authentication-example/ - perhaps a function in there that could send the FCM you wanted as a sort of reflector, so then you could have the example app pull the device token from in-app, call sendFCM from in-app (possibly even with a delay to test backgrounding?) to itself and do the test. Just thinking of how to make this easy to test for the on-average-less-than-1-person-per-day that will test it

Reply:
Ah, that's a good idea! Definitely something I could see being useful for everyone